packages/sdk/wallets/wallet-ledger/package.json

```diff
@@ -29,14 +31,18 @@
     "@celo/wallet-base": "^5.2.0",
     "@celo/wallet-remote": "^5.2.0",
     "@ethereumjs/util": "8.0.5",
-    "@ledgerhq/errors": "^5.50.0",
-    "@ledgerhq/hw-app-eth": "~5.11.0",
-    "@ledgerhq/hw-transport": "~5.11.0",
-    "debug": "^4.1.1"
+    "@ledgerhq/errors": "^6.16.4",
+    "@ledgerhq/hw-app-eth": "^6.36.1",
```

Comment:
since this is patched we should probably lock the version to the exact one patched